flink-libraries/flink-table/src/main/java/org/apache/flink/api/table/functions/utils/MathFunctions.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.api.table.functions.utils;
+
+import java.math.BigDecimal;
+
+public class MathFunctions {
+
+	public static double power(double a, BigDecimal b) {
+		return Math.pow(a, b.doubleValue());
+	}
+
+}
```

Comment:
Scala handles `static` methods a bit differently than Java. Did you try to implement `MathFunctions` as a Scala object? See the paragraph on "Scala’s Sort-of Statics"  in this tutorial: http://www.codecommit.com/blog/scala/scala-for-java-refugees-part-3